swift/Sources/FlatBuffers/FlatBuffersVersion.swift

```diff
@@ -0,0 +1,11 @@
+import Foundation
+
+public enum FlatBuffersVersion {
+    case v1_11_1
+    
+    private static var current: FlatBuffersVersion { return .v1_11_1 }
+    
+    static public func validate(v: FlatBuffersVersion) {
+        assert(v == current)
```

Comment:
This is a runtime check. See how Java does it, it is a compile time check since the version is baked into the name. Please make it work just like the Java version.